active/JWT None Exploit.js

```diff
@@ -4,18 +4,35 @@ var Cookie = Java.type("java.net.HttpCookie");
 var Base64 = Java.type("java.util.Base64");
 var String = Java.type("java.lang.String");
 
-// Exploit information, used for raising alerts
-var RISK = 3;
-var CONFIDENCE = 2;
-var TITLE = "JWT None Exploit";
-var DESCRIPTION =
-  "The application's JWT implementation allows for the usage of the 'none' algorithm, which bypasses the JWT hash verification.";
-var SOLUTION =
-  "Use a secure JWT library, and (if your library supports it) restrict the allowed hash algorithms.";
-var REFERENCE =
-  "https://www.sjoerdlangkemper.nl/2016/09/28/attacking-jwt-authentication/";
-var CWEID = 347; // CWE-347: Improper Verification of Cryptographic Signature
-var WASCID = 15; // WASC-15: Application Misconfiguration
+var ScanRuleMetadata = Java.type(
+  "org.zaproxy.addon.commonlib.scanrules.ScanRuleMetadata"
+);
+
+function getMetadata() {
+  return ScanRuleMetadata.fromYaml(`
+id: 100026
+name: JWT None Exploit
+description: >
+  The application's JWT implementation allows for the usage of the 'none' algorithm,
+  which bypasses the JWT hash verification.
+solution: >
+  Use a secure JWT library, and (if your library supports it) restrict the allowed hash algorithms.
+references:
+  - https://www.sjoerdlangkemper.nl/2016/09/28/attacking-jwt-authentication/
+category: server
+risk: high
+confidence: medium
+cweId: 347
+wascId: 15
```

Comment:
Comments were dropped but in the other script they were kept (don't mind either way just saying for consistency).